### PR TITLE
Add Multi-Language support for Gen 9

### PR DESCRIPTION
**Pull request by xdaybreakerx**
Add multi-language support for Gen 9 Pokemon. 

**Language Support added:**
- Korean
- Russian (Partial - See **Issues** below)
- Thai (Partial - See **Issues** below)
- Chinese (Simplified) 
- Chinese (Traditional)

**Sources:**
- [Korean](https://bulbapedia.bulbagarden.net/wiki/List_of_Korean_Pok%C3%A9mon_names)
- [Russian](https://bulbapedia.bulbagarden.net/wiki/List_of_Russian_Pok%C3%A9mon_names)
- [Thai](https://bulbapedia.bulbagarden.net/wiki/List_of_Thai_Pok%C3%A9mon_names)
- [Chinese (Simplified)](https://bulbapedia.bulbagarden.net/wiki/List_of_Chinese_Pok%C3%A9mon_names)
- [Chinese (Traditional)](https://bulbapedia.bulbagarden.net/wiki/List_of_Chinese_Pok%C3%A9mon_names)

Issues: 
- Source for Russian translation does not include Pokemon 1009-1025 (Future Paradox Pokemon, and Pokemon released in Gen 9 Expansions - The Teal Mask, and The Indigo Disk). Cannot find credible source for translation.
- Source Thai translation does not include Pokemon Pokemon 1011-1025 (Pokemon released in Gen 9 Expansions). Cannot find credible source for translation.

### CODE REVIEW

**Comment by xdaybreakerx** [comment]
Fixes #70

**Comment by sindresorhus** [comment]
> Source for Russian translation does not include Pokemon 1009-1025 (Future Paradox Pokemon, and Pokemon released in Gen 9 Expansions - The Teal Mask, and The Indigo Disk). Cannot find credible source for translation.
Source Thai translation does not include Pokemon Pokemon 1011-1025 (Pokemon released in Gen 9 Expansions). Cannot find credible source for translation.

This needs to be documented in the readme

File: data/zh-hant.json [hunk 1]
```diff
@@ -903,5 +903,127 @@
 	"幽尾玄魚",
 	"大狃拉",
 	"萬針魚",
-	"眷戀雲"
+	"眷戀雲",
+	"新葉喵",
+    "蒂蕾喵",
```
Comment (sindresorhus) [inline data/zh-hant.json:908]: Please follow the existing indentation.

**Comment by xdaybreakerx** [comment]
Screenshot 2024-04-11 at 9 42 18 PM

Updated Readme per request - let me know if you'd like me to change formatting 

Included a screenshot for the indentation issue - on VS Code was showing as correctly indented but obviously not on GitHub. Looks like it should be fixed now!

**Pull request merged**